src/snowflake/connector/cursor.py

```diff
@@ -15,6 +15,7 @@
 import uuid
 import warnings
 from enum import Enum
+from importlib import util
```

Comment:
Looks like I didn’t remove the import that we decided was unnecessary. I’m used to Ruff removing these automatically on save in my projects, sorry about that!

Reply:
could you remove it and add a changelog entry. I'll merge and publish this again

Reply:
I just removed it now, in #1835.

Happy new year!